informer/informer.go

```diff
@@ -98,6 +103,11 @@ func New(config Config) (*Informer, error) {
 	return newInformer, nil
 }
 
+func (i *Informer) Boot(ctx context.Context) error {
+	prometheus.MustRegister(prometheus.Collector(i))
```

Comment:
We don't need to use `Must` now. `func init()` is aways checked, but this is not guaranteed to be ran. I think it's better to `.Register` and return error.

Reply:
Done.